CHANGES.md

```diff
@@ -3,6 +3,8 @@
 ## 1.4.0-pre1
 
 - Add floating IP support (via 1:1 NAT) in Felix.
+- Add tiered security policy based on labels and selectors (PR #979).  Allows
+  for a rich, hierarchical security model.
```

Comment:
I guess there will be a more detailed public presentation of this feature.  When that it exists, it would be good to add the URL here.